docs/source/example_toy_npe_model.md

```diff
@@ -0,0 +1,127 @@
+# Toy Example
+
+The goal of the following tutorial is to take a user from start to finish analyzing GW150914 using dingo.
+
+```{caution}
+This is only a toy example which is useful for testing on a local machine. This
+is NOT meant be used for production gravitational wave analyses.
+```
+
+There are 4 main steps:
+
+1. Generate the waveform dataset
+2. Generate the ASD dataset
+3. Train the network
+4. Do inference
+
+Step 1 Generating a waveform dataset
+------------------------------------
+
+First make a directory for this example where we will store all of our files from this tutorial
+
+```
+cd dingo
+mkdir $(pwd)/toy_npe_model_train_dir
+export TRAIN_DIR=$(pwd)/toy_npe_model_train_dir
+```
+
+To generate a waveform dataset run
+
+```
+dingo_generate_dataset --settings examples/toy_npe_model/waveform_dataset_settings.yaml --out_file $TRAIN_DIR/waveform_dataset.hdf5
+```
```

Comment:
That is true, I think there should be the download from the remote repo instead of the devel install. Though I think setting the $TRAIN_DIR may be convenient, since that way the user can just run the later commands without having to really think about it too much. But yeah having the sed lines later may overcomplicate things. I'll see if I can reorganize things to make it a bit easier. Maybe the file structure could be something like:

toy_npe_model/
   train_dir/
      waveform_dataset.hdf5
      asd_dataset/
         asd_dataset.hdf5
         tmp/
     training/
        train_settings.yaml
        local_settings.yaml
        (SVDs optionally)
        model_latest.pt
   inference/
      GW150914.ini
      outdir_GW150914/
          dingo_pipe_stuff

Reply:
You could flatten it further. Put all the config files from the relevant examples directory at the lowest level. Then put all the training data files in a subdirectory, the trained models in another, and the inference output in the final one:

```
toy_npe_model/

    #  config files
    waveform_dataset_settings.yaml
    asd_dataset_settings.yaml
    train_settings.yaml
    GW150914.ini

    training_data/
        waveform_dataset.hdf5
        asd_dataset.hdf5
        tmp/    #  Contains temporary files from ASD dataset generation

    models/
        model_050.pt
        model_stage_0.pt
        model_latest.pt
        history.txt
        #  etc...

    outdir_GW150914/
        #  dingo_pipe output
```

Reply:
In fact, it might be a good idea to explicitly list that directory structure (as written above) in the documentation for the toy  example. That would be a way to describe all the various files and how they relate to each other.